variables.tf

```
@@ -0,0 +1,65 @@
+##########################
+# IAM User
+##########################
+variable "user_name" {
+  description = "Bitmovin IAM user for Cloud Connect Encoding"
```

```suggestion
  description = "Name of the Bitmovin IAM user for Cloud Connect Encoding"
```
Could be applied to the other name settings here too.

---

Done